mitxgraders/helpers/calc/expressions.py

```diff
@@ -1062,3 +1085,93 @@ def eval_sum(self, parse_result):
             else:
                 raise CalcError("Unexpected symbol {} in eval_sum".format(op))
         return result
+
+PARSER = MathParser()
```

Comment:
This seems somewhat hidden here, given that it's the fundamental object in this module. Perhaps it's worth mentioning in the docstring?